boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -58,7 +58,7 @@
 	pwmleds {
 		compatible = "pwm-leds";
 		red_pwm_led: red_pwm_led {
-			pwms = <&sc_timer 0 PWM_POLARITY_NORMAL>;
+			pwms = <&sc_timer 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
```

Comment:
Also `mimxrt685_evk_cm33` needs to be updated: https://github.com/zephyrproject-rtos/zephyr/blob/dadcb3945a8eb6340262dc815225f61fada5e256/boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts#L73-L83

Reply:
fixed